docs/src/links.md

```diff
@@ -0,0 +1,15 @@
+# Index
+
+Below is the list of types and functions mentioned in the documentation.
+
+## Types
+
+```@index
+Order = [:type]
+```
+
+## Functions
+
+```@index
+Order = [:function]
+```
```

Comment:
I think we can erase these automated sections in favor of the reference guide which is already organized in sections.